packages/components/Checkbox/src/Checkbox.styling.ts

```diff
@@ -33,6 +34,7 @@ export const stylingSettings: UseStylingOptions<CheckboxProps, CheckboxSlotProps
           paddingHorizontal: tokens.paddingHorizontal,
           ...borderStyles.from(tokens, theme),
         },
+        android_ripple: { color: tokens.rippleColor, foreground: true },
       }),
       ['backgroundColor', 'padding', ...borderStyles.keys],
```

Comment:
rippleColor should be added to the cache key

Reply:
Added.

Reply:
Changes completed and approved , hence resolving.